CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to ng-wiz
+
+We would love for you to contribute to ng-wiz and help make it even better than it is
+today! As a contributor, here are the guidelines we would like you to follow:
+
+ - [Code of Conduct](#coc)
+ - [Question or Problem?](#question)
+ - [How to develop ng-wiz](#development)
+   - [Run `ng-wiz` locally](#run-locally)
+   - [Run `ng-wiz` globally](#run-globally)
+ - [Running unit tests](#unit-tests)
+ - [Running end-to-end tests](#ee-tests)
+
+## <a name="coc"></a> Code of Conduct
+Help us keep ng-wiz open and inclusive. Please read and follow our [Code of Conduct][coc].
```

Comment:
Link broken

Reply:
That's because the file it links to is also commited in this PR. It can't link to that file since it doesn't exist until the PR will be merged

Reply:
Take a look at [the rendered version](https://github.com/angular/angular/blob/master/CONTRIBUTING.md) of the angular `CONTRIBUTING.MD` file, and the [raw verions](https://raw.githubusercontent.com/angular/angular/master/CONTRIBUTING.md)

It's the same syntax (it was taken from there), so it should work once the code of conduct is linked to your github repository and that link becomes active